docs/proposals/24-shoot-oidc-issuer.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Shoot OIDC Issuer
+gep-number: 24
+creation-date: 2024-01-09
+status: implementable
+authors:
+- "@dimityrmirchev"
+reviewers:
+- "@vpnachev"
+- "@timuthy"
+- "@rfranzke"
+---
+
+# GEP-24: Shoot OIDC Issuer
+
+## Table of Contents
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+    - [Admission plugin](#admission-plugin)
+    - [Gardenlet](#gardenlet)
+    - [Metadata server](#metadata-server)
+- [Alternatives](#alternatives)
```

Comment:
Not reviewing it, Was reading to know what new changes are inlined in gardener.

The TOC is outdated, `Admission plugin` and `Alternatives` is not present any more in the doc.

Reply:
Thank you, will fix it!